assets/js/app.js

```diff
@@ -1,92 +1,146 @@
-function Book(index, bookTitle, bookAuthor) {
-  this.index = index;
-  this.bookTitle = bookTitle;
-  this.bookAuthor = bookAuthor;
+class Book {
+  constructor(bookTitle, bookAuthor) {
+    this.bookTitle = bookTitle;
+    this.bookAuthor = bookAuthor;
+  }
 }
 
-let books = [];
+let books;
+
+// SavedBooks Class: Handles Storage
+class SavedBooks {
+  static getBooks() {
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
```

Comment:
- [ ] **[OPTIONAL]:** You could achieve the same in one line using any of the [fallback operators](https://salferrarello.com/javascript-or-versus-nullish-coalescing-operator/). As shown below:

```suggestion
    books = JSON.parse(localStorage.getItem('books')) || []
```

This means that if there's a key in LocalStorage called 'books', set the value of the books variable to its content, else fallback to an empty array. 🤓 🛠️